docs/examples/amor.ipynb

```diff
@@ -56,8 +74,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Compute I over Q and the standard deviation of Q\n",
-    "ioq, qstd = pl.compute((NormalizedIofQ, QResolution)).values()"
+    "from ess.reflectometry import save_reference, load_reference\n",
+    "\n",
+    "pl[ReferenceFilePath] = save_reference(pl, './reference.hdf5')\n",
+    "pl.insert(load_reference)"
```

Comment:
Getting `UnsatisfiedRequirement: ('No provider found for type', typing.Optional[ess.reflectometry.types.SampleSize[ess.reflectometry.types.Reference]])` here, and more similar errors below.

Reply:
Odd, the docs build passes. Is it about the sciline version?